docs/source/gbd2019_models/risk_effects/ldl/index.rst

```diff
@@ -105,76 +93,80 @@ Vivarium Modeling Strategy
      - Outcome ID
      - Affected measure
      - Note
-   * -
+   * - Ischemic heart disease
+     - Cause
+     - 493
+     - Mortality and Morbidity (GBD YLLs and YLDs)
+     - Cause-level is mortality only
+   * - Ischemic stroke
+     - Cause
+     - 495
+     - Mortality and Morbidity (GBD YLLs and YLDs)
+     - 
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
      -
+   * - Female only
+     - False
      -
+   * - YLL only
+     - False
      -
+   * - YLD only
+     - False
      -
+   * - Age group start
+     - 10
+     - [25, 29)
+   * - Age group end
+     - 235
+     - [95, 125 years)
 
 Risk Outcome Pair #1
 ++++++++++++++++++++
 
-.. todo::
-
-	Replace "Risk Outcome Pair #1" with the name of an affected entity for which a modeling strategy will be detailed. For additional risk outcome pairs, copy this section as many times as necessary and update the titles accordingly.
-
-.. todo::
+:ref:`See ischemic heart disease documentation <2019_cause_ihd>`
 
-  Link to existing cause model document or other documentation of the outcome in the risk outcome pair.
+The relative risks apply to the incidence rates of acute myocardial infarction and stable angina. They should be applied using the formula affected_measure*(1 – PAF\ :sub:`r367,c493`\) * RR\ :sub:`r367`\). The association was evaluated at the cause level, but the associations should be applied to the incidence rates for both nonfatal components of ischemic heart disease. 
```

Comment:
I was expecting to see the LDL-C exposure in the formula somewhere, e.g.

incidence(i) = incidence*(1-PAF)*RR^{(LDL-C_i - TMREL)^+}